tools/deployment/package/docker-compose.base.yaml

```diff
@@ -0,0 +1,362 @@
+name: "clp-package-base"
+
+# Common service defaults.
+x-service-defaults: &service_defaults
+  image: "${CLP_PACKAGE_CONTAINER_IMAGE_REF:-clp-package}"
+  logging:
+    driver: "local"
+  stop_grace_period: "3s"
```

Comment:
I feel like 3s might be too short for the databases (MariaDB/MySQL and MongoDB) especially when running on hard drives. Can we bump it to 60s for them? Since it's a timeout, it shouldn't slow down the package stop process unless something is very wrong. And if it does, then we're probably causing data loss by using 3s.